src/EnergyPlus/OutputProcessor.cc

```diff
@@ -2955,55 +2741,52 @@ namespace OutputProcessor {
-        Real64 rminValue;
-        Real64 repVal; // The variable's value
         auto &rf = state.dataResultsFramework->resultsFramework;
         auto &sql = state.dataSQLiteProcedures->sqlite;
 
-        repVal = repValue;
-        if (storeType == StoreType::Averaged) {
-            repVal /= numOfItemsStored;
-        }
-        if (repValue == 0.0) {
-            NumberOut = "0.0";
-        } else {
-            NumberOut = format("{:f}", repVal);
-        }
+        Real64 repVal = (storeType == StoreType::Average) ? (StoreValue / NumStored) : StoreValue;
 
         // Append the min and max strings with date information
-        MinOut = fmt::to_string(minValue);
-        MaxOut = fmt::to_string(MaxValue);
-        ProduceMinMaxString(MinOut, minValueDate, freq);
-        ProduceMinMaxString(MaxOut, maxValueDate, freq);
-
         if (rf->timeSeriesEnabled() &&
             (freq == ReportFreq::Day || freq == ReportFreq::Month || freq == ReportFreq::Year || freq == ReportFreq::Simulation)) {
             // add to daily TS data store
-            rf->freqTSData[(int)freq].pushVariableValue(reportID, repVal);
+            rf->freqTSData[(int)freq].pushVariableValue(ReportID, repVal);
         }
 
-        rminValue = minValue;
-        rmaxValue = MaxValue;
         if (sql) {
-            sql->createSQLiteReportDataRecord(reportID, repVal, freq, rminValue, minValueDate, rmaxValue, maxValueDate);
+            sql->createSQLiteReportDataRecord(ReportID, repVal, freq, MinValue, minValueDate, MaxValue, maxValueDate);
         }
 
-        if ((freq == ReportFreq::EachCall) || (freq == ReportFreq::TimeStep) || (freq == ReportFreq::Hour)) { // -1, 0, 1
-            if (state.files.eso.good()) {
-                print(state.files.eso, "{},{}\n", reportID, NumberOut);
+        if (state.files.eso.good()) {
+            std::string NumberOut;
+            if (varType == VariableType::Real) {
+                char tmp[128];
+                dtoa(repVal, tmp);
+                NumberOut = std::string(tmp);
+            } else {
+                // Can someone explain why we are printing integers as
+                // floats and why we are doing it differently than
+                // floats?
+                NumberOut = (repVal == 0.0) ? "0.0" : format("{:f}", repVal);
             }
-        } else { // if ( ( reportingInterval == ReportDaily ) || ( reportingInterval == ReportMonthly ) || ( reportingInterval == ReportSim ) ) {
-                 // // 2, 3, 4
-            if (state.files.eso.good()) {
-                print(state.files.eso, "{},{},{},{}\n", reportID, NumberOut, MinOut, MaxOut);
+            
+            if ((freq == ReportFreq::EachCall) || (freq == ReportFreq::TimeStep) || (freq == ReportFreq::Hour)) { // -1, 0, 1
+                print(state.files.eso, "{},{}\n", ReportID, NumberOut);
+            } else {
+                char minValString[128], maxValString[128];
+                dtoa(MinValue, minValString);
```

Comment:
min/max values are printed using `dtoa` for both real and integer variables. Just to keep things interesting.